staging/cse/windows/provisioningscripts/kubeproxystart.ps1

```diff
@@ -81,13 +86,15 @@ if ($featureGateArgs -ne "") {
     $global:KubeproxyArgList += @("--feature-gates=" + $featureGateArgs)
 }
 
-#
-# cleanup the persisted policy lists
-#
-Import-Module $global:HNSModule
-# Workaround for https://github.com/kubernetes/kubernetes/pull/68923 in < 1.14,
-# and https://github.com/kubernetes/kubernetes/pull/78612 for <= 1.15
-Get-HnsPolicyList | Remove-HnsPolicyList
+if ($IsCleanupHnsNetwork) { # CleanupHnsNetwork is legacy code
+    # cleanup the persisted policy lists
+    Import-Module $global:HNSModule
+    # Workaround for https://github.com/kubernetes/kubernetes/pull/68923 in < 1.14,
+    # and https://github.com/kubernetes/kubernetes/pull/78612 for <= 1.15
+    Get-HnsPolicyList | Remove-HnsPolicyList
```

Comment:
Kube-proxy will delete policylists itself when it is started, so I don't see any reason to delete any HNS resources. It should never be run imho so I am glad you disabled this.